scripts/android.mk

```diff
@@ -1,7 +1,7 @@
-NDK_HOME	:= $(SRCDIR)/$(NDK_DIR)
+ANDROID_NDK_HOME ?= $(SRCDIR)/$(NDK_DIR)
```

Comment:
How about
```
ANDROID_NDK_HOME ?= $(SRCDIR)/$(NDK_DIR)
NDK_HOME := ANDROID_NDK_HOME
```
?

That way we can also pretty easily chain in `ANDROID_NDK_ROOT` without changing all the variable names again if we want to support the `ANDROID_NDK_ROOT/21.lots.of.numbers` in the future as well.

Reply:
OK